### PR TITLE
Fix/shared files move

### PR DESCRIPTION
**Pull request by nettashviro**

### CODE REVIEW

File: src/components/popups/MoveToPopup.vue [hunk 1]
```diff
@@ -54,21 +56,32 @@ export default {
...
 
-      breadcrumbs.push({
-        value: undefined,
-        text: this.$t("pageHeaders.MyDrive"),
-        disabled: !folder,
-      });
+      const getDriveFirstBreadcrumb = (role) =>
+        this.isFolderOwner(role)
+          ? [this.$t("pageHeaders.MyDrive"), false]
+          : [this.$t("pageHeaders.SharedWithMe"), true];
```
Comment (yoavrozner) [inline src/components/popups/MoveToPopup.vue:67]: Why isn't there external network breadcrumbs?

File: src/components/popups/MoveToPopup.vue [hunk 2]
```diff
@@ -91,7 +104,7 @@ export default {
 
       await this.fetchHierachy(folder);
       await this.fetchFolders(folder);
```
Comment (yoavrozner) [inline src/components/popups/MoveToPopup.vue:106]: Promise.all

**Review by yoavrozner — CHANGES_REQUESTED** [review]
Well done!
Please see my notes

File: src/components/popups/MoveToPopup.vue (hunk 1 above)
Comment (nettashviro) [reply src/components/popups/MoveToPopup.vue:67]: you can't move files from external networks

File: src/components/popups/MoveToPopup.vue (hunk 2 above)
Comment (nettashviro) [reply src/components/popups/MoveToPopup.vue:105]: done

File: src/components/popups/MoveToPopup.vue (hunk 1 above)
Comment (yoavrozner) [reply src/components/popups/MoveToPopup.vue:63]: I used `getDriveFirstBreadcrumb` in File.vue. I think we should put it in the utils. What do you think?

Comment (nettashviro) [reply src/components/popups/MoveToPopup.vue:63]: we used this differently. what do you think?

Comment (yoavrozner) [reply src/components/popups/MoveToPopup.vue:63]: Ack

**Review by yoavrozner — APPROVED** [review]
GG WP

**Pull request merged**